### ISSUE TITLE
Add documentation about ways to add custom breaking of words or remove it completely

### ISSUE THREAD

**Issue by Andrew15-5** [typst/typst#5624]
### Description

After watching [the video](https://www.youtube.com/watch?v=WQyqyrRQcSw&t=8m32s), I checked if docs have this:

![screenshot_2024-12-24_17-58-44_inverted](https://github.com/user-attachments/assets/d4af8be8-0f89-4f9b-b6ca-8c0eded1a276)

...It doesn't, so should be added. Actually, some other things from that and other videos should also be documented.

**Comment by xkevio** [comment]
While I don't disagree with making stuff like this more discoverable, it is wrong that this doesn't exist in the docs at all. All of these are listed in the [symbol list](⁠https://typst.app/docs/reference/symbols/sym/) with their shorthand and their full name which is where they should be found, as they are symbols. This is not to say they couldn't appear somewhere else in the docs but the premise is somewhat wrong.

Also, I would kindly ask to change the title of this issue as removing them is not a suitable alternative.

**Comment by Andrew15-5** [comment]
> While I don't disagree with making stuff like this more discoverable, it is wrong that this doesn't exist in the docs at all.

What? I'm pretty sure these should be opposite sentences, but they are the same. This exists in the docs, you said it yourself in the next sentence.

**Comment by xkevio** [comment]
> > While I don't disagree with making stuff like this more discoverable, it is wrong that this doesn't exist in the docs at all.
> 
> What? I'm pretty sure these should be opposite sentences, but they are the same. This exists in the docs, you said it yourself in the next sentence.

No, I only said that I agree that making them easier to find is good *and* that you were wrong in your assumption that it did not exist at all. They exist, yes, but it should be easier to find them.

**Comment by Andrew15-5** [comment]
Ohhhh, I see this now. IMO the wording could be a bit more clear like:

While I don't disagree with making stuff like this more discoverable, it is incorrect to say that this doesn't exist in the docs at all.

Or something along these lines. Or just make them 2 completely separate sentences, because they don't really fit together.

Yeah, I didn't know that it _exists_ at all, because I used the symbols page a few times and for the symbols I looked for I didn't notice any "shorthands", so for me, it was safe to assume this just doesn't exist. So like, I couldn't know it exists, if I didn't see it. And the docs probably never mentions the rare shorthands that are present for some symbols anywhere at all. Not even at the https://typst.app/docs/reference/symbols/sym/ itself.

Moreover, sometimes I just find the symbol and look at how it's spelled and that's it. So I won't even potentially see the shorthand if it exists for a given symbol. Just because I literally have no reason to click on the symbol (most of the time).

**Comment by Andrew15-5** [comment]
There is a list of all shorthands in https://typst.app/docs/reference/symbols/ which is the last match in the search.

![Image](https://github.com/user-attachments/assets/3ab1a54b-7c79-47e2-96b3-c840e9c56b87)
